File: Java/ByteBank/src/ByteBank.java
import br.com.tavernadoshackers.funcionarios.Engenheiro;
import br.com.tavernadoshackers.funcionarios.Gerente;

public class ByteBank {
    public static void main(String[] args) {

        // Instância de Objetos
        Gerente joao = new Gerente("João Marcos", 123123123.70, 0, 21345.6);
        Engenheiro marcos = new Engenheiro("Marcos Santos", 123532343.70, 0, 32345.6);


        // Escopo
        System.out.println();

        joao.recebeSalario();
        marcos.recebeSalario();

        System.out.println();

        joao.transfere(2000, marcos);
        marcos.extrato();
    }
}

File: C#/ByteBank/ByteBank/ByteBank.cs
using System;
using ByteBank.Funcionarios;

namespace ByteBank
{
    static class ByteBank
    {
        static void Main(string[] args)
        {
            Console.WriteLine("|============ ByteBank ============|\n");
            
            // Objetos
            var contaJulio = new Gerente("Júlio Cardoso", 100);
            var contaMarta = new Programador("Marta Costa", 100);
            
            // Escopo
            contaJulio.Transferir(200, contaMarta);
            Console.WriteLine();
            
            contaJulio.Extrato();
            contaMarta.Extrato();
        }
    }
}